src/main.rs

```diff
@@ -272,11 +234,12 @@ fn handle_input_line(kademlia: &mut Kademlia<MemoryStore>, line: String) {
             println!("{}",&key);
             println!("{}",&address);
```

Comment:
These 2 lines are not required, since we are displaying Peerid in `IdentifyEvent`'s `inject_event` method.